src/main.rs

```diff
@@ -1,10 +1,161 @@
+use std::{
+    error::Error,
+    fs::{self},
+    time::Duration,
+};
+
+use api::TendermintClient;
+use clap::Parser;
+use context::Ctx;
+use ibc_client_tendermint::{
+    client_state::ClientState,
+    types::{AllowUpdate, ClientState as ClientStateType, ConsensusState, Header, TrustThreshold},
+};
+
+use ibc_core::{
+    client::context::client_state::{
+        ClientStateCommon, ClientStateExecution, ClientStateValidation,
+    },
+    commitment_types::commitment::{CommitmentPrefix, CommitmentProofBytes, CommitmentRoot},
+    host::types::{
+        identifiers::{ChannelId, ClientId, PortId, Sequence},
+        path::{CommitmentPath, Path},
+    },
+};
+use ibc_core::{
+    client::types::Height, commitment_types::specs::ProofSpecs, host::types::identifiers::ChainId,
+};
+use utils::{base64_to_bytes, fetch_consensus_state, fetch_header, CSReadable};
+
 mod api;
 mod context;
+mod provider;
 mod storage;
+mod utils;
+
+#[derive(Parser, Debug)]
+enum LCCLi {
```

Comment:
what `LCCLi` stands for? Could you add a comment or find better name (eg `CLIcommand`)?

Reply:
A LCCli stands for Light Client Command Line.